samples/blueprint.yaml

```diff
@@ -3,28 +3,35 @@ tosca_definitions_version: cloudify_dsl_1_2
 imports:
   - http://www.getcloudify.org/spec/cloudify/3.3/types.yaml
   - http://www.getcloudify.org/spec/psutil-plugin/1.0/plugin.yaml
+  - http://www.getcloudify.org/spec/aws-plugin/1.3.1/plugin.yaml
```

Comment:
im not sure i understand this - in your other PR, you used your own custom plugin.yaml import; assuming this PR replaces the previous one, shouldnt this be the same?